2-how-node-works/starter/event-loop.js

```diff
@@ -0,0 +1,27 @@
+const fs = require(`fs`);
+const crypto = require(`crypto`);
+
+const start = Date.now();
+process.env.UV_THREADPOOL_SIZE = 4; // setting the number of threads
```

Comment:
Can you consider adding comments on the previous line instead of using inline comments?
We can discuss this offline